src/sass/components/_section--feedback.scss

```diff
@@ -0,0 +1,114 @@
+.section--feedback {
+  margin-bottom: 60px;
+
+  .panel-bar {
+    margin-bottom: 30px;
+
+    .row > * {
+      border-bottom: 4px solid #e2e2e2;
```

Comment:
Using `*` is not the best option. It's a quite inefficient option. Use other selector. You may find proepr elements by className for instance.

Reply:
OK no problem :)

Reply:
done, but this first solution is used everywhere - in sections/components. shouldn't we change it?